l10n_it_fatturapa/__manifest__.py

```diff
@@ -25,7 +25,6 @@
         "l10n_it_vat_payability",
         "l10n_it_fiscal_payment_term",
         "partner_firstname",
-        "l10n_generic_coa",
     ],
```

Comment:
Usato quii https://github.com/tafaRU/l10n-italy/blob/60a4b92797ef59ac8de486bd4e8f29d33edb76c0/l10n_it_fatturapa/demo/account_invoice_fatturapa.xml#L20

Reply:
`account` fa uso di `l10n_generic_coa`
https://github.com/odoo/odoo/blob/e0bafb0715a270b1032647474bde40ca680beff4/addons/account/tests/common.py#L30
ma non dipende da lui.

Tra l'altro i test di questa sono verdi quindi i dati demo sono stati correttamente creati.

Reply:
Ok, in effetti solo l10n_us dipende da l10n_generic_coa ma sembra un leftover.